### ISSUE TITLE
eta parameters are trainable

### ISSUE THREAD

**Issue by backpropper** [pemami4911/REBAR-pytorch#1]
All etas are trainable parameters. You cannot set requires_grad=False for them.

**Comment by pemami4911** [comment]
Right! When I was playing around with the author's TF implementation, I noticed that the eta's were staying very close to 1. So for both the TF implementation and my PyTorch implementation, I experimented with fixing them at 1. From the plot in the README, on the simple binarized MNIST problem, the author's TF implementation of REBAR is still able to do quite well without optimizing the etas. So, I think the issue with my code is something else. I'll add the objective for optimizing the etas to my code soon, though. Thanks!
